### DISCUSSION TITLE
Custom room cleaning

### DISCUSSION THREAD

**Discussion by pvillanyi** [Tasshack/dreame-vacuum#463] [Q&A] [answered]
How do I set the focused room/rooms for a cleaning started from automation?
With the vacuum map card I can select rooms, yes and can start the cleaning...
But I want to create automated cleanings for different rooms.

**Comment by Tasshack** [comment, marked as answer]
https://github.com/Tasshack/dreame-vacuum/blob/master/docs/services.md

**Comment by pvillanyi** [comment]
Hello,

How can I shut the persistent notifications off or set it to another target? Notify.discord or companion app?

Thanks

Peter



A(z) Galaxy eszközömről küldve



-------- Eredeti üzenet --------
Feladó: Tasshack ***@***.***>
Dátum: 2024. 03. 25. 17:20 (GMT+01:00)
Címzett: Tasshack/dreame-vacuum ***@***.***>
Másolatot kap: pvillanyi ***@***.***>, Author ***@***.***>
Tárgy: Re: [Tasshack/dreame-vacuum] Custom room cleaning (Discussion #463)


https://github.com/Tasshack/dreame-vacuum/blob/master/docs/services.md

**Comment by Tasshack** [comment]
By reading all of the documentation before asking questions.

**Comment by pvillanyi** [comment]
In the documentation is just written how I can enable/disable categories in integration configuration.
I want to get these notifications just not as persistent in HA.. It comes on every tablet what is inconvinient... I would like to channel them to discord...
